Gems/Atom/Feature/Common/Assets/Materials/Types/Debug/DebugVertexStreams.azsl

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <viewsrg.srgi>
+#define UvSetCount 2
+#include <Atom/RPI/ShaderResourceGroups/DefaultObjectSrg.azsli>
+#include <Atom/RPI/ShaderResourceGroups/DefaultDrawSrg.azsli>
+#include <Atom/RPI/TangentSpace.azsli>
+
+
+option enum class DebugVertexStream { Normals, Tangents, Bitangents, UVs, TangentW } o_debugVertexStream = DebugVertexStream::Normals;
+option enum class ReconstructionMethods { UseVertexData, UseSurfaceGradient, ReconstructBitangent} o_reconstructionMethod = ReconstructionMethods::UseVertexData;
+
+struct VSInput
+{
+    // Base fields (required by the template azsli file)...
+    float3 m_position : POSITION;
+    float3 m_normal : NORMAL;
+    float4 m_tangent : TANGENT; 
+    float3 m_bitangent : BITANGENT; 
+ 
+    // Extended fields (only referenced in this azsl file)...
+    float2 m_uv0 : UV0;
+};
+
+struct VSOutput
+{
+    // Base fields (required by the template azsli file)...
+    // "centroid" is needed for SV_Depth to compile
+    precise linear centroid float4 m_position : SV_Position;
+    float3 m_normal: NORMAL;
+    float4 m_tangent : TANGENT; 
+    float3 m_bitangent : BITANGENT; 
+    float3 m_worldPosition : UV0;
+
+    // VertexHelper expects the output tangent to be stripped of the w component, so we preserve it here
+    float m_bitangentSign : UV8;
```

Comment:
super nit: just because it hurt my eye, can we move it after the UV1 line or change the order of the UVs so that this gets UV1, and m_uv start at UV2?